.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 orbs:
   aws-s3: circleci/aws-s3@1.0.15
-  browser-tools: circleci/browser-tools@1.4.3
+  browser-tools: circleci/browser-tools@1.4.5
```

Comment:
this should be done in a separated PR, merged and then rebased in all PR instead

Reply:
done in #1466, already merged. So rebase to master, removing your commit.

Reply:
Fixed